Explorer/Assets/Scripts/SceneRunner/ECSWorld/ECSWorldFactory.cs

```diff
@@ -17,6 +17,8 @@
 using ECS.Unity.EngineInfo;
 using ECS.Unity.Systems;
 using System.Collections.Generic;
+using SystemGroups.Visualiser;
+using UnityEditor;
```

Comment:
also need to be surrounded by `#if UNITY_EDITOR`

Reply:
You don't need `using UntiyEditor` at all